src/ORM/Behavior/TreeBehavior.php

```diff
@@ -437,7 +437,7 @@ public function childCount(EntityInterface $node, bool $direct = false): int
      * @return \Cake\ORM\Query
      * @throws \InvalidArgumentException When the 'for' key is not passed in $options
      */
-    public function findChildren(Query $query, array $options): Query
+    public function findChildren(Query $query, array $options = []): Query
```

Comment:
This implies that the changes here are going to require updates to all application custom finders as well.